documentation/docs/guide/GettingStarted/deploy_locally.md

```diff
@@ -10,134 +11,125 @@ All development setup is done with a Mac OS
 
 Install brew if not already installed.
 
-- Install mkcert tool for creating certificates ```brew install mkcert```
-- Only if using **Firefox**, Install nss```brew install nss```
-- Create a self signed Certificate Authority ```mkcert -install```
+- Install mkcert tool for creating certificates `brew install mkcert`
+- Only if using **Firefox**, Install nss`brew install nss`
+- Create a self signed Certificate Authority `mkcert -install`
 
-## Setup
-### Download Source Code
+## Setup Local Repository and certs
 
-```git clone https://github.com/bcgov/gdx-agreements-tracker.git```
+### Download Source Code
 
+`git clone https://github.com/bcgov/gdx-agreements-tracker.git`
 
 ### Setup Certificates
-**All bash commands are assumed from root of repository** ```./gdx-agreements-tracker```
+
+**All bash commands are assumed from root of repository** `./gdx-agreements-tracker`
 A self signed certificate is required for local development deployments.
-- in a terminal window go to the GDX agreements tracker repository ```cd gdx-agreements-tracker```
-- create a ```.cert``` directory in the frontend folder if not already created ```mkdir -p ./frontend/.cert```
+
+- in a terminal window go to the GDX agreements tracker repository `cd gdx-agreements-tracker`
+- create a `.cert` directory in the frontend folder if not already created `mkdir -p ./frontend/.cert`
   - This is where your certificates will be saved.
   - This folder is also excluded from the repository via the .gitignore file.
-- create a key and cert file ```mkcert -key-file ./frontend/.cert/key.pem -cert-file ./frontend/.cert/cert.pem "localhost"```
+- create a key and cert file `mkcert -key-file ./frontend/.cert/key.pem -cert-file ./frontend/.cert/cert.pem "localhost"`
 
+## Set up and Deploy Database
 
-## Setup and Deploy Database
 The database is a postgres database, which needs to be deployed via docker compose.
+
 - Add a `./backend/.env` by either creating a new file or using [sample.env](https://github.com/bcgov/gdx-agreements-tracker/blob/development/backend/sample.env) as the template
-- Update/add all database [environment variables]
-```sh 
+- Update/add all database environment variables
+
+```sh
 #/backend/.env
 ...
+DATABASE_AUTO_DEPLOY=1
+POSTGRES_DB=gat_db
+POSTGRES_HOST=0.0.0.0
```

Comment:
not sure the reason you changed from localhost -> 0.0.0.0, not exactly the same, hopefully this doesn't cause some weird failure.

Reply:
works on my machine
I don't know why it would be 0.0.0.0 anyway?